### PR TITLE
Carthage support

### PR DESCRIPTION
**Pull request by RomanPodymov**
Hello.
Thank you for DTTJailbreakDetection.
In this pull request I added support for Carthage.

### CODE REVIEW

File: Sources/DTTJailbreakDetection/DTTJailbreakDetection.m [hunk 1]
```diff
@@ -20,7 +20,7 @@
 //  CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
-#import "DTTJailbreakDetection.h"
+#import "include/DTTJailbreakDetection.h"
```
Comment (thii) [inline Sources/DTTJailbreakDetection/DTTJailbreakDetection.m]: Please add `include` to the header search paths instead.

Comment (RomanPodymov) [reply Sources/DTTJailbreakDetection/DTTJailbreakDetection.m]: Hello @thii 
It works even without any header search paths changes, tested with Carthage 0.34.0.

**Review by thii — APPROVED** [review]

**Comment by ralbertini** [comment]
I tried to use with Carthage, but I got this error on carthage update:

[13:48:51]: ▸ *** Skipped building DTTJailbreakDetection due to the error:
[13:48:51]: ▸ Dependency "DTTJailbreakDetection" has no shared framework schemes for any of the platforms: iOS
[13:48:51]: ▸ If you believe this to be an error, please file an issue with the maintainers at https://github.com/thii/DTTJailbreakDetection/issues/new

:/

**Comment by RomanPodymov** [comment]
Hello @ralbertini 
Could you please show the content of your Cartfile?

**Comment by ralbertini** [comment]
github "AFNetworking/AFNetworking" == 3.1.0
github "Mantle/Mantle"
github "reidmain/FDKeychain"
github "michaeltyson/TPKeyboardAvoiding"
github "SnapKit/Masonry"
github "DavydLiu/DLRadioButton"
github "danielgindi/Charts" ~> 3.4.0
github "TakeScoop/SwiftyRSA"
github "thii/DTTJailbreakDetection"

github "SDWebImage/SDWebImage"
github "evgenyneu/keychain-swift" ~> 18.0
github "HeroTransitions/Hero"
github "malcommac/SwiftDate"

github "rechsteiner/Parchment"
github "pocketsvg/PocketSVG"
github "fjcaetano/ReCaptcha"

I run after again and it worked 
*** Checking out DTTJailbreakDetection at "0.4.0"

but it not appears on ../Carthage/Build/iOS/ folder as a .framework

Should I add files to project from ../Carthage/Checkouts/DTTJailbreakDetection/ manually?

**Comment by RomanPodymov** [comment]
Hello @ralbertini 
DTTJailbreakDetection with Carthage support was not released. Replace `github "thii/DTTJailbreakDetection"` with `github "thii/DTTJailbreakDetection" "master"` in your Cartfile and run `carthage update DTTJailbreakDetection`.

**Pull request merged**